projects/skaffold.dev/package.yml

```diff
@@ -0,0 +1,38 @@
+# https://skaffold.dev/
+
+distributable:
+  url: https://github.com/GoogleContainerTools/skaffold/tree/v{{version}}
+
+versions:
+  github: GoogleContainerTools/skaffold/tags
+
+platforms:
+  - darwin
+  - linux
+
+dependencies:
+
+build:
+  dependencies:
+    curl.se: '*'
+  working-directory: ${{prefix}}
+  script: |
+    curl -Lfo skaffold curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v{{version}}/skaffold-$PLATFORM
+    # old releases are not having github release assets so using storage.googleapis.com for more reliability
+    #curl -Lfo skaffold https://github.com/GoogleContainerTools/skaffold/releases/download/v{{version}}/skaffold-$PLATFORM
+    chmod +x skaffold
+    mkdir -p bin
+    mv skaffold bin
```

Comment:
If this just downloading a blob into place then:

- distributable should just be ~ at the top level.
- it should have a vendored warning key.
```yaml
distributable: ~

warnings:
  - vendored
```